distribution/server/licenses/LICENSE-flatbuffer.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
If it's apache license, we don't need to include it again. We have a section for all apache license dependencies in in the `LICENSE.bin.txt` file